assets/runtime/functions

```diff
@@ -791,7 +791,7 @@ configure_redmine() {
 
   # configure secure-cookie if using SSL/TLS
   if [[ ${REDMINE_HTTPS} == true ]]; then
-    exec_as_redmine sed -i "s/:key => '_redmine_session'/:secure => true, :key => '_redmine_session'/" ${REDMINE_INSTALL_DIR}/config/application.rb
+    exec_as_redmine sed -i "/:secure => true, :key => '_redmine_session'/! s/:key => '_redmine_session'/:secure => true, :key => '_redmine_session'/" ${REDMINE_INSTALL_DIR}/config/application.rb
```

Comment:
I don't consider this to be an ugly fix or even a hack.  Here you are preventing sed from appending yet another `secure:` token if it already exists.

LGTM

Reply:
Linking to my comments here: https://github.com/sameersbn/docker-redmine/issues/297#issuecomment-343500736

Reply:
It is somewhat ugly in that it requires a pattern where the "secure => ..." must be directly in front of the "key" part..
If this repository or upstream redmine add the other cookie flag ("httponly" or "http_only" depending on version of ruby rails) this might fail.. Unfortunately I did not have time to fix this better, perhaps I will give a try during the weekend.

Reply:
I think its fine.  The original sed looks for `:key => '_redmine_session'` and replaces it with `:secure => true, :key => '_redmine_session'` so your check for `:secure => true, :key => '_redmine_session'` will always find the ":secure" token that this sed added. Unless we add another sed that appends another key using this approach, but at least it would be our fault it breaks and not upstream Redmine.

The original sed is a bit fragile if Redmine removes the `_redmine_session` key, however.